preprocessors/tif-decompress.preprocessor.js

```diff
@@ -0,0 +1,43 @@
+var gdal = require('gdal');
+
+module.exports = function(infile, outfile, callback) {
+  var ds;
+  var dscopy;
+  var driver = gdal.drivers.get('GTiff');
+  outfile = outfile + '.tif';
```

Comment:
@GretaCB when `.tif` is added here then the file is saved with a `.tif`. But I'm seeing the resulting `outfile` value reported at https://github.com/mapbox/preprocessorcerer/blob/master/bin/perform-preprocessorcery.js#L21 as not having a `.tif` extension. Is that expected or a problem?